src/interface/src/app/plan/create-scenarios/create-scenarios.component.ts

```diff
@@ -207,12 +216,12 @@ export class CreateScenariosComponent implements OnInit {
         }
         // setting treatment question
         if (scenario.configuration.treatment_question) {
-          this.prioritiesComponent.setFormData(
+          this.prioritiesComponent?.setFormData(
             scenario.configuration.treatment_question
           );
         }
         // setting constraints
-        this.constraintsPanelComponent.setFormData(scenario.configuration);
+        this.constraintsPanelComponent?.setFormData(scenario.configuration);
```

Comment:
same here, I dont think we should add this optional chaining

Reply:
yep, I was focused on removing them from spec files, I forgot I changed these too. I suspect this is going to cause more failing tests, but let me try to make another change to the test configurations to address all of these issues.
I don't think any of the options are going to be simple, but we'll see.